integration_testing/features/coach/coach-can-edits-group.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Edit a group’s name
+    Coach should be able to sign in to access Kolibri Coach tab
+    Coach can edit a group's name
+
+  Background:
+    Given you are in coach groups page
+    Given that there are groups created
```

Comment:
I was thinking that it might be good to adopt the strategy of listing the `Background` "givens" in order to cover the more general conditions **first** (`there are groups created`, meaning that the groups should be created even before the user arrived to the groups page), and the more specific ones (`I am on coach groups page`) **second**.

I would also recommend we keep the pronouns consistent, so if `Scenario` steps are written from the point of view `I`, the "givens" in the `Background` should be too. I now realized that in my own previous PR I used `you` everywhere 🤦‍♀️, and will have to correct that, as `I` seems to be the general consensus...